src/modules/webhooks/WebhooksModule.ts

```diff
@@ -44,8 +45,11 @@ export default class WebhooksModule extends ConnectorRuntimeModule<WebhooksModul
 
         try {
             this.logger.debug(`Sending request to webhook '${url}' for trigger '${trigger}'.`);
-
-            const response = await this.axios.post(url, payload, { headers: webhook.target.headers });
+            const correlationId = correlator.getId();
```

Comment:
where does this correlationId comes from? as far as I see it it will always be undefined.

Reply:
Nop the webhook module will be triggered from the runtime events and therefor is most of the time in a call stack of an usecase.

Reply:
have you tested this?

Reply:
it should be easy to also test this using tests as we already use webhooks for eventing